### ISSUE TITLE
Transcripts are read with the locale codepage on Windows — non-ASCII text is corrupted in training data

### ISSUE THREAD

**Issue by Mramaran** [Amanlabh/footprint#2]
### Description

`parse_session` opens transcript files without an explicit encoding
(`footprint_core/transcripts.py`, line 32):

```python
for line in open(path, errors="replace"):
```

Claude Code transcripts are UTF-8 JSONL, but on Windows `open()` defaults to the
ANSI codepage (typically cp1252). Every multi-byte character — arrows and
box-drawing in tool output, `…`, non-English text, emoji — is decoded wrong, and
`errors="replace"` masks the damage instead of surfacing it. The mojibake flows
straight into `data/train.jsonl`, so the model is fine-tuned on corrupted text.
Windows is a documented, supported platform for `collect`.

### Reproduction (Windows)

1. Have any Claude Code session containing non-ASCII output (most tool results qualify).
2. Run `footprint collect`.
3. Inspect `data/train.jsonl`: UTF-8 sequences appear as `ГÇ£`-style garbage or `�` replacements.

### Suggested fix

```python
for line in open(path, encoding="utf-8", errors="replace"):
```

While at it, audit the other `open()` calls in the package (e.g. writing
`train.jsonl`/`valid.jsonl` in `collect`, reading it in `service.status`) and pass
`encoding="utf-8"` everywhere for consistency — output is currently safe only by
accident (`json.dumps` defaults to `ensure_ascii=True`). Happy to send a PR with a
test fixture containing non-ASCII content.
